assets/console-plugin/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+---
+# Source: openshift-console-plugin/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: monitoring-plugin
+  namespace: openshift-monitoring
+  labels:
+    app: monitoring-plugin
+    app.kubernetes.io/name: monitoring-plugin
+    app.kubernetes.io/instance: monitoring-plugin
+    app.kubernetes.io/part-of: monitoring-plugin
+    app.openshift.io/runtime-namespace: monitoring-plugin
```

Comment:
out of curiosity, what's the purpose of this label? I suppose that it's required by the console server?

Reply:
@kyoto, could you please help answer this?

Reply:
`app.openshift.io/runtime-namespace` does not appear to be used, so let's remove it.